File: velolabs_repos/lattis_android/app/src/main/java/com/lattis/ellipse/presentation/ui/base/activity/BaseBackWhiteArrowActivity.java
package com.lattis.ellipse.presentation.ui.base.activity;

import android.os.Bundle;
import androidx.annotation.Nullable;

/**
 * Created by Velo Labs Android on 17-04-2017.
 */

public abstract class BaseBackWhiteArrowActivity<Presenter extends ActivityPresenter> extends BaseActivity<Presenter> {

    private final int ARROW_WITH_HALF_WHITE = 02;

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

    }

    @Override
    protected void configureViews() {
        super.configureViews();
        setupAppbarCloseIcon(toolbar, ARROW_WITH_HALF_WHITE);
    }

    @Override
    protected void onResume() {
        super.onResume();


    }


}

File: velolabs_repos/lattis_android_v2/app/src/main/java/com/lattis/lattis/presentation/base/activity/BaseBackWhiteArrowActivity.kt
package com.lattis.lattis.presentation.base.activity

import android.os.Bundle
import com.lattis.lattis.presentation.base.BaseView
import com.lattis.lattis.presentation.ui.base.activity.BaseActivity
import kotlinx.android.synthetic.main.view_toolbar.*

/**
 * Created by Velo Labs Android on 17-04-2017.
 */
abstract class BaseBackWhiteArrowActivity<Presenter : ActivityPresenter<V>,V:BaseView> :
    BaseActivity<Presenter, V>() {
    private val ARROW_WITH_HALF_WHITE = 2
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
    }

    override fun configureViews() {
        super.configureViews()
        setupAppbarCloseIcon(toolbar, ARROW_WITH_HALF_WHITE)
    }

    override fun onResume() {
        super.onResume()
    }
}